src/main/java/frc/robot/commands/ShootCommand.java

```diff
@@ -49,13 +45,13 @@ public void execute() {
     if(shooter.flywheelUpToSpeed(shooterPositions.getShootSpeed() * 0.875)){
         indexer.setIndexerSpeed(Constants.INDEXER_FEED_SPEED);
         intake.setIntakeSpeed(0.5);
+        shooter.launchCommand().withTimeout(0.5).schedule();
```

Comment:
LaunchCommand causes a runtime error with real target, add logic to only run launchCommand in Sim mode